samtranslator/model/sam_resources.py

```diff
@@ -1796,6 +1797,7 @@ class SamStateMachine(SamResourceMacro):
     PermissionsBoundary: Optional[Intrinsicable[str]]
     AutoPublishAlias: Optional[PassThrough]
     DeploymentPreference: Optional[PassThrough]
+    UseAliasAsEventTarget: Optional[bool]
```

Comment:
More of a design question. Thoughts on this going under the `Events` property [here](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-property-statemachine-statemachineeventsource.html) rather than being at the top level? Since this property doesn't really make sense to have without `Events` also being defined

Reply:
I agree with Seb's argument.